cmd/archive/release/flag.go

```diff
@@ -0,0 +1,40 @@
+package release
+
+import (
+	"github.com/Masterminds/semver/v3"
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	nameFlag     = "name"
+	providerFlag = "provider"
+	releasesFlag = "releases"
```

Comment:
Let's stick to convetion:

```suggestion
	flagName     = "name"
	flagProvider = "provider"
	flagReleases = "releases"
```

Reply:
Didn't know, thanks.